client/data/emails/types.ts

```diff
@@ -0,0 +1,38 @@
+export type EmailAccountEmail = {
+	domain: string;
```

Comment:
It may be a good idea to be more specific about what that `domain` refers to, even if that's obvious from the type here. I know we had this problem in the backend where it wouldn't be easy to determine if it was a domain object or just a domain name, so we ended up using a more descriptive name:

```suggestion
	domain_name: string;
```

Reply:
Well if that data comes from the backend, it's too late to change that 😄.

Reply:
You're right – it comes from the back-end.